WooCommerce/Classes/POS/ViewModels/ItemListViewModel.swift

```diff
@@ -150,6 +161,5 @@ private extension ItemListViewModel {
             value: "Retry",
             comment: "Text for the button appearing on the item list screen when there's an error loading products."
         )
-        static let firstPageNumber: Int = 1
```

Comment:
nit: since this is specific to the WC API (starting at 1 whereas it might start at 0 for other API), it might be nice having this as a constant. WDYT of keeping this constant and setting `currentPage`'s default value (in two places - variable declaration and `reload`) to `firstPageNumber - 1`?

Reply:
Yes, this is next on my [list](https://github.com/woocommerce/woocommerce-ios/issues/14186), since I want to make a clear separation between first fetch vs subsequent fetches. 

At the moment we call `populatePointOfSaleItems()` both when the view first appears, as well as in subsequent fetches for pagination, so right now fetching the 1st page correctly is a bit of a side-effect of the internal function count and relies that we start on 0. Ideally I think we should pass the page into the function as a parameter and keep it stateless, but was a bit of too much refactoring right now to keep this PR small.